ftwhttp/request.go

```diff
@@ -103,8 +104,14 @@ func (r *Request) AddHeader(name string, value string) {
 // AddStandardHeaders adds standard headers to the request, if they don't exist
 //
 // This will add Content-Length and the proper Content-Type
-func (r *Request) AddStandardHeaders(size int) {
-	r.headers.AddStandard(size)
+func (r *Request) AddStandardHeaders() {
+	// For better performance, we always close the connection (unless otherwise)
+	r.headers.Add("Connection", "close")
+
+	// If there is data or POST method header, we add the length also
+	if len(r.data) > 0 || r.requestLine.Method == "POST" {
```

Comment:
Define `methodsWithBodyRegex` as a constant with `regexp.MustCompile("^POST|PUT|PATCH|DELETE$")`.

```suggestion
	if len(r.data) > 0  || methodsWithBodyRegex.matchString(r.requestLine) {
```

Reply:
Updated by https://github.com/coreruleset/go-ftw/pull/135/commits/a6e2e79d88d9b9d7d6904bd5c131a9b7aab54e6a